Migrations/20201227_AddTeamsTable.cs

```diff
@@ -0,0 +1,34 @@
+using FluentMigrator;
+
+namespace CribblyBackend.Migrations
+{
+    [Migration(202012271812)]
+    public class AddTeamsTable : Migration
+    {
+        public override void Down()
+        {
+            Delete.Table("Players");
+            Delete.Table("Teams");
+        }
+
+        public override void Up()
+        {
+            Create.Table("Teams")
```

Comment:
Didn't the teams table already exist?

Reply:
Deleted this file. I wasn't quite sure on how the migrations worked, but I got it now!